doc/audio.md

```diff
@@ -13,75 +22,75 @@ flutter:
     - assets/audio/music.mp3
 ```
 
-Then simply run:
+Then you have the following methods at your disposal:
 
 ```dart
-import 'package:flame/flame.dart';
+import 'package:flame_audio/flame_audio.dart';
 
-Flame.audio.play('explosion.mp3');
-Flame.audio.playLongAudio('music.mp3');
-```
+// For shorter reused audio clips, like sound effects
+FlameAudio.play('explosion.mp3');
 
-Or, if you prefer:
+// For looping an audio file
+FlameAudio.loop('music.mp3');
 
-```dart
-import 'package:flame/flame_audio.dart';
+// For playing a longer audio file
+FlameAudio.playLongAudio('music.mp3');
 
-FlameAudio audio = FlameAudio();
+// For looping a longer audio file
+FlameAudio.loopLongAudio('music.mp3');
 
-audio.play('explosion.mp3'); // Or
-audio.playLongAudio('music.mp3');
+// For background music that should be paused/played when the pausing/resuming the game
+FlameAudio.bgm.play('music.mp3');
 ```
+The difference between the `play/loop` and `playLongAudio/loopLongAudio` is that `play/loop` makes
+use of optimized features that allow sounds to be looped without gaps between their iterations, and
+almost no drop on the game frame rate will happen. You should whenever possible, prefer these
+methods.
 
-The difference is that each instance shares a different cache. Normally you would want to use the `Flame.audio` instance and totally share the cache.
-
-You must have an appropriate folder structure and add the files to the `pubspec.yaml` file, as explained above.
-
-It can be an MP3, OGG or a WAV file (those are the ones I tested and it worked)
+`playLongAudio/loopLongAudio` allows for audios of any length to be played, but they do create frame
+rate drop, and the looped audio will have a small gap between iterations.
 
-This uses the [audioplayers](https://github.com/luanpotter/audioplayer) lib, in order to allow playing multiple sounds simultaneously (crucial in a game). You can check the link for more in-depth explanations.
+You can use [the `Bgm` class](bgm.md) (via `FlameAudio.bgm`) to play looping background music
+tracks. The `Bgm` class lets Flame manage the auto pausing and resuming of background music tracks
+when pausing/resuming the game.
```

Comment:
it's a bit unclear here what `when pausing/resuming the game`, that refers to app foregrounded/backgrounded right? might be useful to specify that